pycloudlib/ec2/cloud.py

```diff
@@ -161,7 +161,10 @@ def _get_name_for_image_type(self, release: str, image_type: ImageType, daily: b
             return f"ubuntu-pro-server/images/{disk_type}/ubuntu-{release}-{release_ver}-*"
 
         if image_type == ImageType.PRO_FIPS:
-            return f"ubuntu-pro-fips*/images/{disk_type}/ubuntu-{release}-{release_ver}-*"
+            return f"ubuntu-pro-fips-server*/images/{disk_type}/ubuntu-{release}-{release_ver}-*"
```

Comment:
I checked that adding `-server` here still matches the name string of all the existing Pro FIPS images.

Reply:
Looking at the others it seems the `*` here was to support fips/fips updates as a single type. Given the separation between fips and fips-updates now I think the `*` is unnecessary. All strict FIPS images are `ubuntu-pro-fips-server/...` and all fips-updates images are `ubuntu-pro-fips-updates-server`

Reply:
done!